checks/containers/container_engine/xccl_tests.py

```diff
@@ -0,0 +1,129 @@
+# Copyright 2024 Swiss National Supercomputing Centre (CSCS/ETH Zurich)
+# ReFrame Project Developers. See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+import pathlib
+import sys
+
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+sys.path.append(str(pathlib.Path(__file__).parent.parent.parent / 'mixins'))
+
+from container_engine import ContainerEngineMixin  # noqa: E402
+
+
+class XCCLTestBase(rfm.RunOnlyRegressionTest, ContainerEngineMixin):
+    valid_prog_environs = ['builtin']
+    sourcesdir = None
+    test_name = parameter(['all_reduce', 'sendrecv'])
+    num_nodes = variable(int, value=2)
+    min_bytes = variable(str, value='1024M')
+    max_bytes = variable(str, value='1024M')
+    container_env_table = {
```

Comment:
do we want to add this as an option to ContainerEngineMixin (or else where) so it doesn't get repeated everywhere?